app/views/simplified-account/settings/card-payments/index.njk

```diff
@@ -0,0 +1,100 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Card payments
+{% endblock %}
+
+{% block settingsContent %}
+  <h1 class="govuk-heading-l">Card payments</h1>
+  <p class="govuk-body">GOV.UK Pay uses 3D Secure for all card payments. 3D Secure (3DS) adds an extra layer of
+    authentication to user payments.</p>
+
+
+{{ govukSummaryList({
+  classes: "govuk-!-margin-bottom-9",
+  rows: [
+    {
+      key: {
+        text: "Collect billing address"
+      },
+      value: {
+        text: collectBillingAddressEnabled | boolToOnOrOff
+      },
+      actions: {
+        items: [
+          {
+            href: collectBillingAddressLink,
+            text: "Change",
+            classes: "govuk-link--no-visited-state",
+            visuallyHiddenText: "collect billing address"
+          }
```

Comment:
these can be conditionally rendered if the change link is provided by the controller